rpc_jobs/rpc_aio.yml

```diff
@@ -70,7 +84,20 @@
 
 - job-template:
     # DEFAULTS
-    STAGES: "Allocate Resources, Connect Slave, Prepare Deployment, Deploy RPC w/ Script, Setup MaaS, Verify MaaS, Install Tempest, Tempest Tests, Cleanup, Destroy Slave"
+    STAGES: >-
+      Allocate Resources,
+      Connect Slave,
+      Prepare Deployment,
+      Deploy RPC w/ Script,
+      Setup MaaS,
+      Verify MaaS,
+      Install Tempest,
+      Tempest Tests,
+      Prepare Kibana Selenium,
+      Kibana Tests,
```

Comment:
As this is the default list, it will effect PRs, do you know how much time these two add? I noticed that test build 80 took ~4 hrs rather than ~3:15. Run time is variable though, especially if waiting for cloud quota.